### ISSUE TITLE
Mysql error for db tenant username.

### ISSUE THREAD

**Issue by jartaud** [hyn-me/multi-tenant#12]
Env: Homestead (Windows 8.1. I'll try on my nix box).

Hello, when i issue the command below:

```batchfile
sudo php artisan multi-tenant:setup --tenant=mysupertenant
--email=admin@mysupertenant.com --hostname=mysupertenant.com --webserver=apache
```

it throws:

```batchfile
[PDOException]
  SQLSTATE[HY000]: General error: 1470 String '1-mysupertenant-com' 
is too long for user name (should be no longer than 16)
```

and running 
```batchfile
sudo php artisan multi-tenant:setup --tenant=EXAMPL
--email=foo@example.com --hostname=exampl.com --webserver=apache
```

I got:
```batchfile
[ErrorException]
  is_dir() expects parameter 1 to be a valid path, array given
```

**Comment by Luceos** [comment]
Thanks the first issue is caused by the unixname being too long, I will fix that asap. The second issue about is_dir I need to take a look at.

**Comment by Luceos** [comment]
Got the second issue fixed and committed to latest dev-master. Now unto issue 1.

**Comment by Luceos** [comment]
Issue 1 is fixed in 4a9659694c1b6d3801e7e60177390a78e48f51ea
requires tag version up for some packages; todo, until then closed
